classes/File.js

```diff
@@ -216,4 +224,14 @@ class HomepageType {
   }
 }
 
-module.exports = { File, PageType, CollectionPageType, ResourcePageType, ResourceType, ImageType, DocumentType, DataType, HomepageType }
+class MenuType {
+  constructor() {
+    this.folderName = '_data/'
+  }
+  getFolderName() {
+    return this.folderName
+  }
+}
```

Comment:
There already exists a type that does this - `DataType`. My bad I did not read the file entirely

Reply:
Fixed by removing the redundant `MenuType` in [1f684](https://github.com/isomerpages/isomercms-backend/commit/1f684ab53451464b91a53d5a6e027c3211cc3075)